examples/board_ds1338/ds1338demo.c

```diff
@@ -35,7 +35,7 @@ ds1338_virt_t ds1338_virt;
 
 int main(int argc, char *argv[])
 {
-	elf_firmware_t f;
+	elf_firmware_t f = {{0}};
```

Comment:
In c99 you can just specify = {} -- no need of the zeroes.

Reply:
I can not really recall. why '0' is there.  I was puzzled by existing {{}} in run_avr.c and perhaps I thought it looked better with at least some explicit initialisation.  Do you consider it a reason to delay integration?